[2023-07-09T07:13:15.282+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 1_init_once_seed_data.load_seed_data_once manual__2023-07-09T07:03:55.361926+00:00 [queued]>
[2023-07-09T07:13:15.288+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 1_init_once_seed_data.load_seed_data_once manual__2023-07-09T07:03:55.361926+00:00 [queued]>
[2023-07-09T07:13:15.288+0000] {taskinstance.py:1308} INFO - Starting attempt 3 of 3
[2023-07-09T07:13:15.296+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): load_seed_data_once> on 2023-07-09 07:03:55.361926+00:00
[2023-07-09T07:13:15.300+0000] {standard_task_runner.py:57} INFO - Started process 440 to run task
[2023-07-09T07:13:15.303+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '1_init_once_seed_data', 'load_seed_data_once', 'manual__2023-07-09T07:03:55.361926+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/init.py', '--cfg-path', '/tmp/tmpqiwjn9rr']
[2023-07-09T07:13:15.304+0000] {standard_task_runner.py:85} INFO - Job 47: Subtask load_seed_data_once
[2023-07-09T07:13:15.341+0000] {task_command.py:410} INFO - Running <TaskInstance: 1_init_once_seed_data.load_seed_data_once manual__2023-07-09T07:03:55.361926+00:00 [running]> on host 8f42e64974a7
[2023-07-09T07:13:15.417+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='1_init_once_seed_data' AIRFLOW_CTX_TASK_ID='load_seed_data_once' AIRFLOW_CTX_EXECUTION_DATE='2023-07-09T07:03:55.361926+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-09T07:03:55.361926+00:00'
[2023-07-09T07:13:15.418+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-09T07:13:15.419+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt && dbt seed --profiles-dir .']
[2023-07-09T07:13:15.427+0000] {subprocess.py:86} INFO - Output:
[2023-07-09T07:13:16.819+0000] {subprocess.py:93} INFO - [0m07:13:16  Running with dbt=1.5.2
[2023-07-09T07:13:17.324+0000] {subprocess.py:93} INFO - [0m07:13:17  Registered adapter: snowflake=1.5.1
[2023-07-09T07:13:17.465+0000] {subprocess.py:93} INFO - [0m07:13:17  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-07-09T07:13:17.465+0000] {subprocess.py:93} INFO - There are 2 unused configuration paths:
[2023-07-09T07:13:17.466+0000] {subprocess.py:93} INFO - - models.dbt.analysis
[2023-07-09T07:13:17.466+0000] {subprocess.py:93} INFO - - models.dbt.transform
[2023-07-09T07:13:17.485+0000] {subprocess.py:93} INFO - [0m07:13:17  Found 5 models, 1 test, 0 snapshots, 0 analyses, 438 macros, 0 operations, 4 seed files, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-07-09T07:13:17.489+0000] {subprocess.py:93} INFO - [0m07:13:17
[2023-07-09T07:13:18.468+0000] {subprocess.py:93} INFO - [0m07:13:18  Concurrency: 200 threads (target='dev')
[2023-07-09T07:13:18.470+0000] {subprocess.py:93} INFO - [0m07:13:18
[2023-07-09T07:13:18.511+0000] {subprocess.py:93} INFO - [0m07:13:18  1 of 4 START seed file public.bookings_1 ....................................... [RUN]
[2023-07-09T07:13:18.512+0000] {subprocess.py:93} INFO - [0m07:13:18  2 of 4 START seed file public.bookings_2 ....................................... [RUN]
[2023-07-09T07:13:18.514+0000] {subprocess.py:93} INFO - [0m07:13:18  3 of 4 START seed file public.customers ........................................ [RUN]
[2023-07-09T07:13:18.515+0000] {subprocess.py:93} INFO - [0m07:13:18  4 of 4 START seed file public.library_transaction .............................. [RUN]
[2023-07-09T07:13:20.706+0000] {subprocess.py:93} INFO - [0m07:13:20  4 of 4 OK loaded seed file public.library_transaction .......................... [[32mINSERT 20[0m in 2.18s]
[2023-07-09T07:13:20.910+0000] {subprocess.py:93} INFO - [0m07:13:20  1 of 4 OK loaded seed file public.bookings_1 ................................... [[32mINSERT 7[0m in 2.39s]
[2023-07-09T07:13:21.053+0000] {subprocess.py:93} INFO - [0m07:13:21  2 of 4 OK loaded seed file public.bookings_2 ................................... [[32mINSERT 7[0m in 2.53s]
[2023-07-09T07:13:21.178+0000] {subprocess.py:93} INFO - [0m07:13:21  3 of 4 OK loaded seed file public.customers .................................... [[32mINSERT 2[0m in 2.66s]
[2023-07-09T07:13:21.266+0000] {subprocess.py:93} INFO - [0m07:13:21
[2023-07-09T07:13:21.267+0000] {subprocess.py:93} INFO - [0m07:13:21  Finished running 4 seeds in 0 hours 0 minutes and 3.78 seconds (3.78s).
[2023-07-09T07:13:21.281+0000] {subprocess.py:93} INFO - [0m07:13:21
[2023-07-09T07:13:21.282+0000] {subprocess.py:93} INFO - [0m07:13:21  [32mCompleted successfully[0m
[2023-07-09T07:13:21.283+0000] {subprocess.py:93} INFO - [0m07:13:21
[2023-07-09T07:13:21.284+0000] {subprocess.py:93} INFO - [0m07:13:21  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2023-07-09T07:13:22.039+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-09T07:13:22.063+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=1_init_once_seed_data, task_id=load_seed_data_once, execution_date=20230709T070355, start_date=20230709T071315, end_date=20230709T071322
[2023-07-09T07:13:22.109+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-09T07:13:22.121+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
